src/OpenApi/HTTP.hs

```diff
@@ -0,0 +1,62 @@
+module OpenApi.HTTP where
```

Comment:
@mbj Afaict, `module`s with an empty export list export all the contained functions. I see that this `module` only defines a single function that will be exported, but I wonder if, for maintenance reasons, we should start nailing the exports down precisely? The `OpenApi.HTTP` module is maybe not the most convincing module for this flag, but I chose to flag here as this is the first green line occurrence in this commit where I noticed this, the `OpenApi.Paths` module is also "affected".

At this point I'm mainly curious if we already have an established rule wrt such default exports? I tend to think that relying on exporting everything by default may lead to undesired exports as the codebase grows?

Reply:
@snusnu I do not consider the import / export list mature here at all. Potentially I'll not expose all but `OpenApi` root and do not export internal modules at all.

For now I only added export lists to things where:

* I do not need everything exported (this is AFAIK a case, and re-specifying everything where the default is everything: is just redundant).
* I'am sure about the future public interface.

This is not "mature" code. It needs to grow into our usual standards.